codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/SymbolVisitor.java

```diff
@@ -80,12 +86,79 @@
                 .build();
 
         escaper = ReservedWordSymbolProvider.builder()
-                // TODO: escape reserved member names
-                .nameReservedWords(reservedWords)
+                .nameReservedWords(reserveInterfaces(model))
+                .memberReservedWords(reservedWords)
                 // Only escape words when the symbol has a definition file to
                 // prevent escaping intentional references to built-in types.
                 .escapePredicate((shape, symbol) -> !StringUtils.isEmpty(symbol.getDefinitionFile()))
                 .buildEscaper();
+
+        // Reserved words that only apply to error members.
+        ReservedWords reservedErrorMembers = new ReservedWordsBuilder()
+                .put("ErrorCode", "ErrorCode_")
+                .put("ErrorFault", "ErrorMessage_")
+                .put("Unwrap", "Unwrap_")
+                .put("Error", "Error_")
+                .build();
```

Comment:
I think `ErrorFault` and `ErrorMessage` might be an incorrect here.

Reply:
whoops